contributors/design-proposals/architecture/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -158,8 +155,82 @@ The template for a KEP is precisely defined in the [template proposal][]
 
 [template proposal]: https://github.com/kubernetes/community/pull/1124
 
+### KEP Metadata
+
+There is a place in each KEP for a YAML document that has standard metadata.
+This will be used to support tooling around filtering and display.  It is also
+critical to clearly communicate the status of a KEP.
+
+Metadata items:
+* **kep-number** Required
+  * Each proposal has a number.  This is to make all references to proposals as
+    clear as possible.  This is especially important as we create a network
+    cross references between proposals.
+  * Before having the `Accepted` status, the number for the KEP will be in the
+    form of `draft-YYYYMMDD`.  The `YYYYMMDD` is replaced with the current date
+    when first creating the KEP.  The goal is to enable fast parallel merges of
+    pre-acceptance KEPs.
+  * On acceptance a sequential dense number will be assigned.  This will be done
+    by the editor and will be done in such a way as to minimize the chances of
+    conficts.  The final number for a KEP will have no prefix.
+* **title** Required
+  * The title of the KEP in plain language.  The title will also be used in the
+    KEP filename.  See the template for instructions and details.
+* **status** Required
+  * The current state of the KEP.
+  * Must be one of `Draft`, `Deferred`, `Accepted`, `Rejected`, `Withdrawn`,
```

Comment:
I'm fine with iterating on these stages in subsequent PRs, but I still think the "Accepted" stage is unnecessarily inconsistent with existing project terminology, which would be "Approved".

Reply:
I'm not picky.  Changed to Approved.